### PR TITLE
[iOS] Initial WunderLINQ Support

### PR DESCRIPTION
**Pull request by kconger**
I'm one of the developers of the WunderLINQ. The WunderLINQ enables you to use your motorcycle's handlebar controls to control your smartphone. Essentially the device acts as a Bluetooth keyboard.
Here is a video of the integration in action: [https://youtu.be/2QHUiheVF5c](https://youtu.be/2QHUiheVF5c)

Support was [recently](https://github.com/mapsme/omim/pull/12378) committed to the Android App, this PR is for the iOS App.

Hi @beloal, @alexzatsepin mentioned I should seek guidance from you on adding support for a "back_url" argument to deep links.  

I noticed "backurl" is already used so I've used that instead for the iOS implementation.  Please let me know if you would prefer a different approach.

### CODE REVIEW

**Comment by kconger** [comment]
Hi @beloal, @alexzatsepin 

I was wondering if you had some time coming up to review.  A few of our mutual users keep asking me :) . They are excited about MAPS.ME support.  Thanks for your time.

**Comment by kconger** [comment]
Any chance that this may get reviewed soon?  Thank you for your time.

File: iphone/Maps/Core/DeepLink/DeepLinkHandler.swift [hunk 1]
```diff
@@ -105,6 +105,14 @@ class DeepLinkURL {
     isLaunchedByDeeplink = false
     deeplinkURL = nil
   }
+  
+  func getBackUrl() -> String {
+    if deeplinkURL != nil {
```
Comment (beloal) [inline iphone/Maps/Core/DeepLink/DeepLinkHandler.swift]: Maybe 
`guard let urlString = deeplinkURL?.url.absoluteString else { return "" }
`?
Force unwrap on the next line isn't safe

**Comment by kconger** [comment]
Hi @beloal I've made your suggested changes.  I'm not sure why GitHub says there is a conflict for the two files.

Let me know if you want me to make a clean fork and new PR.  I can't seem to dig myself out of this git hole. :)

**Comment by kconger** [comment]
Hi @beloal,  Do you have any more feedback on this pull request?

**Comment by kconger** [comment]
Hi @beloal, any time to review?  Thanks for your time.

**Comment by beloal** [comment]
Hi, @kconger. I can't merge the PR due to conflicts. Could you please resolve them? Thanks

**Comment by kconger** [comment]
Hi, @beloal thanks for reviewing.  GitHub is currently showing "no conflicts with the base branch".   Not sure why maybe something hasn't updated on my end?

**Comment by beloal** [comment]
@kconger could you just try to rebase your development branch on top of master?

**Comment by kconger** [comment]
@beloal working on it right now. I'll reply back once complete.

**Comment by beloal** [comment]
@kconger I can see that there is 11 commits in this PR already. Maybe it would be better to open a new PR with the relevant changes only?

**Comment by kconger** [comment]
@beloal I can redo the PR to keep things cleaner.

**Comment by kconger** [comment]
@beloal Here is the new [one](https://github.com/mapsme/omim/pull/13198)

**Pull request closed without merging**